### PR TITLE
Resource permit UX

### PR DESCRIPTION
**Pull request by dividedmind**

### CODE REVIEW

File: lib/conjur/command/resources.rb [hunk 1]
```diff
@@ -68,6 +68,7 @@ class Conjur::Command::Resources < Conjur::Command
         id = full_resource_id( require_arg(args, "RESOURCE") )
         role = require_arg(args, "ROLE")
         privilege = require_arg(args, "PRIVILEGE")
+        $stderr.print "Granting #{privilege} on #{id} to #{role}... "
```
Comment (kgilpin) [inline lib/conjur/command/resources.rb]: The message should read : role, privilege, resource ; to be consistent

Comment (dividedmind) [inline lib/conjur/command/resources.rb]: I assume you mean RESOURCE ROLE PRIVILEGE ;)

**Comment by dividedmind** [comment]
The order in the message now reflects the cmdline; btw, the cmdline order is highly unamenable to constructing a sentence from, which might be why we tend to misremember it. (I was worried it was my doing, but apparently not: 5c4b81f4 ;) )

If you can come up with a better message, please do change it!

**Comment by dividedmind** [comment]
A friendly reminder :)

**Comment by dividedmind** [comment]
I'm just going to merge it next week if there's no more comments.

**Comment by kgilpin** [comment]
It doesn't match the way we present things.

Role - privilege - resource

**Comment by dividedmind** [comment]
Well, I can change it to that, but I think if it matches the command line order, it will make it easier to learn.

**Comment by jjmason** [comment]
Well just to weigh in...it should obviously be `"Grant role #{role} permission to #{privilege} resource #{resource}."`, right?

**Comment by kgilpin** [comment]
I believe role, privilege, resource is the right order for the message.

**Comment by dividedmind** [comment]
Yes, that's what Kevin's saying; I just thought if the message reflected the command line order it will make people automatically remember it. But I don't have strong opinion, just any message will do; without a message it tends to silently do something you didn't expect.

**Comment by kgilpin** [comment]
`resource permit` is deprecated.

**Pull request closed without merging**